RELEASING.md

```diff
@@ -5,6 +5,7 @@
 - Run `fastlane bump_and_update_changelog version:X.Y.Z` (where X.Y.Z is the new version) to update the version number in `android/build.gradle`, `android/gradle.properties` and `PurchasesHybridCommon.podspec`
 - Update purchases-android version in `android/build.gradle`
 - Update purchases-ios pod version in `PurchasesHybridCommon.podspec` and `ios/PurchasesHybridCommon/Podfile`.
+- run `pod lib lint` to ensure that no problems are found with the pod. 
```

Comment:
Let's make sure we add this as part of CI at some point

Reply:
+100

Reply:
added to backlog